codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSwiftDependency.kt

```diff
@@ -3,68 +3,42 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 package software.amazon.smithy.aws.swift.codegen
-import software.amazon.smithy.codegen.core.SymbolDependency
-import software.amazon.smithy.swift.codegen.Dependency
-import software.amazon.smithy.swift.codegen.resources.Resources
+import software.amazon.smithy.swift.codegen.SwiftDependency
 
-enum class AWSSwiftDependency(
-    override val target: String,
-    private val branch: String?,
-    private val version: String,
-    private val url: String,
-    private val localPath: String,
-    override var packageName: String
-) : Dependency {
-    AWS_SDK_IDENTITY(
-        "AWSSDKIdentity",
-        "main",
-        "0.1.0",
-        "https://github.com/awslabs/aws-sdk-swift",
-        Resources.computeAbsolutePath("aws-sdk-swift", "", "AWS_SDK_SWIFT_CI_DIR"),
-        "aws-sdk-swift"
-    ),
-    AWS_SDK_HTTP_AUTH(
-        "AWSSDKHTTPAuth",
-        "main",
-        "0.1.0",
-        "https://github.com/awslabs/aws-sdk-swift",
-        Resources.computeAbsolutePath("aws-sdk-swift", "", "AWS_SDK_SWIFT_CI_DIR"),
-        "aws-sdk-swift"
-    ),
-    AWS_SDK_EVENT_STREAMS_AUTH(
-        "AWSSDKEventStreamsAuth",
-        "main",
-        "0.1.0",
-        "https://github.com/awslabs/aws-sdk-swift",
-        Resources.computeAbsolutePath("aws-sdk-swift", "", "AWS_SDK_SWIFT_CI_DIR"),
-        "aws-sdk-swift"
-    ),
-    AWS_CLIENT_RUNTIME(
-        "AWSClientRuntime",
-        "main",
-        "0.1.0",
-        "https://github.com/awslabs/aws-sdk-swift",
-        Resources.computeAbsolutePath("aws-sdk-swift", "", "AWS_SDK_SWIFT_CI_DIR"),
-        "aws-sdk-swift"
-    ),
-    AWS_COMMON_RUNTIME(
-        "AwsCommonRuntimeKit",
-        "main",
-        "0.1.0",
-        "https://github.com/awslabs/aws-crt-swift",
-        "",
-        "aws-crt-swift"
-    );
+class AWSSwiftDependency {
 
-    override fun getDependencies(): List<SymbolDependency> {
-        val dependency = SymbolDependency.builder()
-            .putProperty("target", target)
-            .packageName(packageName)
-            .version(version)
-            .putProperty("url", url)
-            .putProperty("branch", branch)
-            .putProperty("localPath", localPath)
-            .build()
-        return listOf(dependency)
+    companion object {
```

Comment:
Changed this file from an enum to a set of static val's.  Updated the paths to point to locations within the SDK project.